tasks/terraform-cli/src/tests/steps/task-context.steps.ts

```diff
@@ -160,6 +161,27 @@ export class TaskContextSteps {
         this.ctx.skipExistingWorkspace = true;
     }
 
+    @given("subcommand is {string}")
+    public subCommand(subCommand: string){
+        this.ctx.workspaceSubCommand = subCommand;

```

Comment:
The tests are failing because i think this should be setting `this.ctx.stateSubCommand`

```suggestion
        this.ctx.stateSubCommand = subCommand;
```